flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/expressions/ScalarFunctionsTest.scala

```diff
@@ -4195,4 +4195,48 @@ class ScalarFunctionsTest extends ScalarTypesTestBase {
       "f55=f57",
       "true")
   }
+
+  //-------------------------------------------------------------------
+  // JSON functions
+  //-------------------------------------------------------------------
+  @Test
+  def testJsonExists(): Unit = {
```

Comment:
These tests actually test only SQL expressions, not table api expressions. Have you ever tried to run those expressions from Table API?

Reply:
> These tests actually test only SQL expressions, not table api expressions. Have you ever tried to run those expressions from Table API?

@wuchong  recommends just SQL  expressions first.

Reply:
Do you mean supporting these json functions in SQL first? Or just testing for SQL?
If the answer is supporting these json functions in SQL firstly, I prefer we do not add these table api related expressions for now, which are not tested for now.

Reply:
> I prefer we do not add these table api related expressions for now,

You are right, Let me do not add these table api related expressions for now.